corehq/apps/app_manager/models.py

```diff
@@ -4290,7 +4274,6 @@ def validate_jar_path(self):
     @property
     def jad_settings(self):
         settings = {
-            'JavaRosa-Admin-Password': self.admin_password,
```

Comment:
I think you can remove this whole property (and likely some nearby code, like `validate_jar_path` - basically anything relating to JADs or JARs).